libraries/AP_RCTelemetry/AP_CRSF_Telem.h

```diff
@@ -173,13 +173,46 @@ class AP_CRSF_Telem : public AP_RCTelemetry {
         uint8_t payload[57];   // largest possible frame is 60
     } PACKED;
 
+    // Frame to hold passthrough telemetry
+    struct PassthroughFrame {
+        uint8_t sub_type;
+        uint16_t appid;
+        uint32_t data;
+    } PACKED;
+
+    // Frame to hold passthrough telemetry
+    struct PassthroughArrayFrame {
+        uint8_t sub_type;
+        uint8_t size;
+        struct {
+            uint16_t appid;
+            uint32_t data;
+        } PACKED frames[9];
```

Comment:
Maybe define a constant for this somewhere

Reply:
done